Artkal palettes

davidnilssonart opened the pull request:
Added palettes for four of Artkals lines of beads that loads when Perler
is started.

davidnilssonart committed:
Artkal palettes

Added palettes for four of Artkals lines of beads that loads when Perler
is started.

dolkow commented on src/se/dolkow/imagefiltering/app/gui/configuration/PaletteSettings.java:
```diff
@@ -71,7 +71,7 @@ public PaletteSettings(AbstractReduceColorsFilter f) throws XMLParserException,
 		
 		XMLParser xmlp = XMLParser.getInstance();
-		for (String s : new String[]{"hama-builtin.xml", "nabbi-builtin.xml", "perlerbeads-builtin.xml"}) { //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
+		for (String s : new String[]{"hama-builtin.xml", "nabbi-builtin.xml", "perlerbeads-builtin.xml", "artkal-serie-s-5mm-hard.xml", "artkal-serie-r-5mm-soft.xml", "artkal-serie-c-2.6mm-hard.xml", "artkal-serie-a-2.6mm-soft.xml"}) { //$NON-NLS-1$ //$NON-NLS-2$ //$NON-NLS-3$
```
Please add corresponding NON-NLS comments for the constant strings (otherwise, Eclipse will nag me about them).

Should look like the rest, just with higher numbers. :)

dolkow reviewed (changes requested):
I know this is your first git commit ever, so I'm going to give you a learning opportunity! There is a small change I'd like you to make before accepting this commit.

davidnilssonart committed:
Added NON-NLS

Added NON-NLS.
Also, the added Artkal palettes are only for solid colors, not neon,
translucent and others.
